cmd/tkn/main.go

```diff
@@ -26,7 +29,25 @@ func main() {
 	tp := &cli.TektonParams{}
 	tkn := cmd.Root(tp)
 
-	if err := tkn.Execute(); err != nil {
+	args := os.Args[1:]
+	cmd, _, _ := tkn.Find(args)
```

Comment:
Why `tkn.Find(args)` doesn't return an error when an invalid command is passed? 
I tried with another CLI `Find` returns err when the command is invalid, it happens only with `tkn`. :smile:

Reply:
@SM43 my guess is the `suggest` part but I am not sure :sweat: